jobs/test-runner.jpl

```diff
@@ -126,16 +117,9 @@ node("docker && test-runner") {
             sh(script: "rm -rf ${jobs_dir}")
 
             timeout(time: 15, unit: 'MINUTES') {
-                parallel(
-                    kci_core: {
-                        j.cloneKciCore(
-                            kci_core,
-                            params.KCI_CORE_URL, params.KCI_CORE_BRANCH)
-                    },
-                    artifacts: {
-                        getArtifacts(artifacts)
-                    },
-                )
+                artifacts: {
+                    getArtifacts(artifacts)
+                },
```

Comment:
Probably should be }?

Reply:
Yes, Jenkins complained about that too.  In fact it just needs to be `getArtifacts(artifacts)`.  Fixed now and tested on staging.